Source/Core/VideoCommon/AbstractGfx.h

```diff
@@ -101,6 +101,10 @@ class AbstractGfx
   // m_backbuffer_height may change after this function returns.
   virtual void BindBackbuffer(const ClearColor& clear_color = {}) {}
 
+  // In case we temporarily set another buffer as render target,
+  // we can use this to bind the backbuffer again.
+  virtual void ReBindBackbuffer() {}
```

Comment:
Not sure about adding this interface.  Can we not just acquire the current one with `GetCurrentFramebuffer()` and then reapply it?

Reply:
Mmm yeah, gold point, I could probably cache it before changing it... Didn't really think about it.